.circleci/config.yml

```diff
@@ -0,0 +1,154 @@
+aliases:
+  - &credentials
+    name: Retrieve temporary Algolia credentials if needed
+    command: |
+      if [ -z "ALGOLIA_ADMIN_KEY_MCM" ]; then
+        curl -s https://algoliasearch-client-keygen.herokuapp.com | sh >> $BASH_ENV
+      fi
```

Comment:
Maybe we want to move this to just before we run the tests? This way, any errors with bundler surface quicker, and we don't download the keys unless we're actually going to run the tests, if that makes sense 🙂